### ISSUE TITLE
It seems that current code only realize the reward loss, how about the pretrain loss mentioned in the paper? Do the reward loss and pretrain loss are trained sequentially?

### ISSUE THREAD

**Issue by lkh329** [THUDM/ImageReward#24]

**Comment by WuJie1010** [comment]
Very nice work! I have the same problem. I only confuse that why the grad scale is 1e-3 and only train 100 steps? thanks a lot.
